.github/workflows/unittest.yml

```diff
@@ -24,3 +24,95 @@ jobs:
           git submodule sync --recursive && git submodule update --init --recursive
           ./scripts/setup-ubuntu.sh 
           make release EXTRA_CMAKE_FLAGS="-DVELOX_ENABLE_ARROW=ON"
+
+  gluten-test:
+    runs-on: self-hosted
+    steps:
+      - name: Setup docker container
+        run: |
+          git clone https://github.com/oap-project/gluten && cd gluten
+          #git config --global alias.pr '!f() { git fetch -fu ${2:-origin} refs/pull/$1/head:pr/$1 && git checkout pr/$1; }; f'
+          BRANCH=$(echo ${GITHUB_REF#refs/heads/})
+          sed -i 's/oap-project/${GITHUB_REPOSITORY_OWNER}/g' ep/build-velox/src/get_velox.sh
+          sed -i 's/VELOX_BRANCH=main/VELOX_BRANCH=${BRANCH}/g' ep/build-velox/src/get_velox.sh
+          EXTRA_DOCKER_OPTIONS="--name velox-backend-ubuntu2004-test-$GITHUB_RUN_ID -e NUM_THREADS=30 --detach" \
```

Comment:
Can the `GITHUB_RUN_ID` be possible same with gluten test at the same time?
Do we have enough resources to do the test? Now we always add the unit test in gluten repo to test velox code

Reply:
> Can the `GITHUB_RUN_ID` be possible same with gluten test at the same time?

the ID should be unique AFAIK - I'm reading below docs from GH:
https://docs.github.com/en/actions/learn-github-actions/variables


> Do we have enough resources to do the test? Now we always add the unit test in gluten repo to test velox code

currently there's only one runner - we could add more testing resources to check. Indeed this patch tries to save the time on creating the "empty patch" on gluten. I also found there are a few velox patches merged with out triggering gluten tests.